lib/schema.ts

```diff
@@ -840,21 +846,23 @@ const RawConfigSchema = z
       })
       .optional(),
     deployRag: z.boolean().optional().default(false),
+    deployChat: z.boolean().optional().default(false),
+    deployUi: z.boolean().optional().default(false),
     logLevel: z.union([z.literal('DEBUG'), z.literal('INFO'), z.literal('WARNING'), z.literal('ERROR')]),
     lambdaConfig: lambdaConfigSchema,
     lambdaSourcePath: z.string().optional().default('./lambda'),
-    authConfig: AuthConfigSchema,
+    authConfig: AuthConfigSchema.optional(),
```

Comment:
can we also add a validation for the authconfig being defined if we do end up using the chat stacks? The UI won't work otherwise in its current state (it'll deploy, but we won't be able to use it)